packages/content-conformance/README.md

```diff
@@ -0,0 +1,111 @@
+# `@hashicorp/platform-content-conformance`
+
+A system to execute checks against HashiCorp's various types of content. Very similar to [ESLint](https://eslint.org/), but tailored to checking content (MDX) and data (JSON, YAML) instead of JavaScript source.
+
+## Installation
+
+```shell
+$ npm install @hashicorp/platform-content-conformance
+```
+
+## Usage
+
+### CLI
+
+_The CLI is in progress._
+
+```shell
+$ hc-content [options] [path/to/file] [path/to/file]
+
+Options:
+  --cwd
+  --config ./path/to/conformance.config.js
+```
+
+### JavaScript
+
+The package exposes its underlying functionality through the `ContentConformanceRunner` class.
+
+```js
+import { ContentConformanceRunner } from '@hashicorp/platform-content-conformance'
+
+async function main() {
+  const runner = new ContentConformanceRunner()
+
+  await runner.init()
+
+  await runner.run()
+
+  console.log(runner.report())
+}
+
+main()
+```
+
+## Configuration
+
+On its own, the content conformance system does not know where the files to be checked are located, or what rules should be used. To configure the checker, we create a `content-conformance.config.js` file located in your project's root directory.
+
+```js
+export default {
+  root: '.',
+  contentFileGlobPattern: 'content/**/*.mdx',
+  rules: {
+    'no-h1': 'error',
+  },
+}
+```
+
+## Rules
+
+The core value of the conformance checker lies in the rules that you configure. The core package ships with a number of rules and presets that can be used, and custom rules can also be created in your project. A rule is a JavaScript module that exports a specific object format.
+
+```js
+// ./rules/my-rule.js
+
+/** @type {import('@hashicorp/platform-content-conformance').Rule} */
```

Comment:
Would this be, or extend `.ConformanceRuleBase`?

I'm not seeing a plain `Rule` type, so just checking in case I'm missing something obvious

Reply:
Ah yeah, this was future thinking so it doesn't exist yet.

Reply:
Cool, no issues there!

And I may have missed mention of this in your video, but are rules under `src/rules/*` deemed _"built-in"_ rules? (i.e. Will there be built-in, and custom external rules?)

Reply:
Yeah, that's correct! I think for now we'll explicitly support built-in rules and custom rules in projects, but since we're just loading the module with `import()` we could in theory pass a path to something in an external package.